docker-images/kafka-base/Dockerfile

```diff
@@ -9,34 +9,34 @@ ENV KAFKA_HOME=/opt/kafka
 RUN groupadd -r -g 1001 kafka && useradd -r -m -u 1001 -g kafka kafka
 
 # Set Scala and Kafka version
-ENV SCALA_VERSION=2.11
```

Comment:
From the [download page of Apache Kafka](https://kafka.apache.org/downloads)

> We build for multiple versions of Scala. This only matters if you are using Scala and you want a version built for the same Scala version you use. Otherwise any version should work (2.11 is recommended)

What's the motivation to change to `.12` , @scholzj  ?

Reply:
@matzew To use the same as downstream. I'm not aware of any issues with Scala 2.12. Are you aware of anything?

Reply:
downstream as in ? 

I just find it weird to use .12 - when the Kafka team recommends .11 :smile: